routes/api/children.js

```diff
@@ -1,7 +1,8 @@
 const childrenController = require('../../controllers/childrenController');
 
 module.exports = (app) =>{
-    app.post('/api/createChild',childrenController.createChild);
+    app.post('/api/AddChild',chilrenController.createChild);
+    // app.post('/api/createChild',childrenController.createChild);
```

Comment:
Is this route going to be use in the future? I assume so just throw on a TODO: or FIXME:

Reply:
Yes, we are planning for future use. Will add the TODO or FIXME